api/envoy/config/listener/v3/listener_components.proto

```diff
@@ -109,6 +110,163 @@ message FilterChainMatch {
     EXTERNAL = 2;
   }
 
+  // Matches filter chains by the destination port.
```

Comment:
These comments all say "Matches ...", but that's a little misleading.  These don't actually match anything by themselves; they just indicate to the matching framework what value to use as input.  So maybe change the comments to say something like "Matcher input that allows matching connections on the destination port."

Reply:
Done.